### PR TITLE
[fix][modify][i3991] unread message filter

### PR DESCRIPTION
**Pull request by Reinhard-sheng**
@victormartinelicocorp 
@elicoidal

### CODE REVIEW

**Review by martintorresvm — CHANGES_REQUESTED** [review]
@Reinhard-sheng LGTM to be tested.
Just small details:
Bump the module version number
Add yourself as collaborator on README 

@elicoidal could we add @Reinhard-sheng on our CLA

**Comment by Reinhard-sheng** [comment]
@victormartinelicocorp 
Please help check the version number.

File: business_requirement/__manifest__.py [hunk 1]
```diff
@@ -6,7 +6,7 @@
     'category': 'Business Requirements Management',
     'summary': 'Manage the Business Requirements (stories, scenarios, gaps\
         and test cases) for your customers',
-    'version': '10.0.1.1.0',
+    'version': '10.0.1.1.1',
```
Comment (elicoidal) [inline business_requirement/__manifest__.py]: You need to update the module so next number should be: 10.0.1.2.0

**Review by elicoidal — CHANGES_REQUESTED** [review]
small detail then we merge

**Review by martintorresvm — APPROVED** [review]

**Comment by Reinhard-sheng** [comment]
change version again.
@elicoidal 
@victormartinelicocorp

**Review by elicoidal — APPROVED** [review]
LGTM

**Comment by elicoidal** [comment]
CLA done

**Review by lonelysun — APPROVED** [review]

**Pull request merged**